Sources/Sentry/SentryUIViewControllerSwizziling.m

```diff
@@ -13,6 +13,14 @@
 #if SENTRY_HAS_UIKIT
 #    import <UIKit/UIKit.h>
 
+/**
+ * UIApplication must conform to SentryUIApplication
+ * in order to swizzle work.
+ */
+@interface
+UIApplication (SentryUIApplication) <SentryUIApplication>
+@end
```

Comment:
`l`: I don't understand why we need this class. Can you explain it, please?

Reply:
This is not a class, is an extension. Now 'SentryUIViewControllerSwizzling' does not relies on UIApplication any more, it just need some object that conforms to 'SentryUIApplication'. This way we can create tests with mocks, because thats no way to create an instance of UIApplication in tests.

With this extensions I'm just specifying that UIApplication conforms to SentryUIApplication.

Reply:
Can you please add this info as a comment, please?

Reply:
Sure